samples/sample_hevc_fei/include/fei_buffer_allocator.h

```diff
@@ -53,7 +53,7 @@ class FeiBufferAllocator
         if (!request.Width || !request.Height) throw mfxError(MFX_ERR_NOT_INITIALIZED, "BufferAllocRequest is uninitialized for buffer");
 
         mfxU32 va_num_elem = 1;
-        mfxU32 va_data_size = sizeof(*(T::Data));
+        mfxU32 va_data_size = sizeof(buffer->Data);
```

Comment:
no! this code will retrieve size of pointer (Data is not an array!), so it will be 8!

change it to sizeof(buffer->Data[0]);